OpenEdXMobile/src/test/java/org/edx/mobile/view/WhatsNewFragmentTest.java

```diff
@@ -0,0 +1,77 @@
+package org.edx.mobile.view;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import org.edx.mobile.BuildConfig;
+import org.edx.mobile.R;
+import org.edx.mobile.util.FileUtil;
+import org.edx.mobile.util.Version;
+import org.edx.mobile.whatsnew.WhatsNew;
+import org.edx.mobile.whatsnew.WhatsNewFragment;
+import org.edx.mobile.whatsnew.WhatsNewModel;
+import org.junit.Test;
+import org.robolectric.shadows.support.v4.SupportFragmentTestUtil;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.text.ParseException;
+import java.util.List;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+public class WhatsNewFragmentTest extends UiTest {
```

Comment:
Moving the `getWhatsNewMessages` function to a Utility class will rid this class of this inheritance.

Reply:
We need a context in **FileUtil.loadTextFileFromResources(context, R.raw.whats_new)**
os it must be a UI test. But i have refactored it to Util class and extending it from BaseTest for code simplicity.